webapp/src/components/chat/chat-list/ChatList.tsx

```diff
@@ -133,19 +133,15 @@ export const ChatList: FC = () => {
 
     useEffect(() => {
         // Ensure local component state is in line with app state.
-        if (filterText !== '') {
-            // Reapply search string to the updated conversations list.
-            const filteredConversations: Conversations = {};
-            for (const key in conversations) {
-                if (conversations[key].title.toLowerCase().includes(filterText.toLowerCase())) {
-                    filteredConversations[key] = conversations[key];
-                }
+        const filteredConversations: Conversations = {};
```

Comment:
These are conversations that are filtered IN, right?  Not conversations that are filtered OUT?

If so, a better name might be conversationsToDisplay or something similar since filteredConversations seems to imply that the conversations are filtered OUT, which is misleading.